scenarios/python/console/premium_speech_demo/README.md

```diff
@@ -0,0 +1,41 @@
+# Instructions to run Speech + LLM Samples
```

Comment:
Hi @changlong-liu , `premium_speech_demo` does not look like a self-explaining folder name. What about AOAI_rewriting as the folder name?

For the README title, What about `Scenario: Continuous Speech Recognition and Rewriting via Azure OpenAI`?